backends/dpdk/run-dpdk-ptf-test.py

```diff
@@ -81,21 +85,12 @@
 
 
 # Check if target ports are ready to be connected (make sure infrap4d is on)
-def is_port_alive(host, port) -> bool:
-    try:
-        with socket.create_connection((host, port), timeout=5) as conn:
-            return True
-    except (socket.timeout, ConnectionRefusedError):
-        return False
-
-
-# We want to disable IPv6 to avoid ICMPv6 spam
-def diable_IPv6():
-    testutils.exec_process("sysctl -w net.ipv6.conf.all.disable_ipv6=1")
-    testutils.exec_process("sysctl -w net.ipv6.conf.default.disable_ipv6=1")
-    # Also filter igmp packets, -w is necessary because of a race condition
-    testutils.exec_process("iptables -w -A OUTPUT -p 2 -j DROP")
-    return testutils.SUCCESS
+def is_port_alive(ns, port) -> bool:
```

Comment:
Is there a more pythonic way to do this? Something where we do not have to call netstat?

For example, like this check:
https://github.com/p4lang/p4c/blob/main/tools/testutils.py#L134

Reply:
We have utilities for this. 
`bridge.ns_exec` should take care of executing in the right namespace.

Reply:
`ns_exec` simply appends `ip netns exec <ns>` before the cmd. It cannot handle calling python functions in an easy way.

Reply:
Oh I see what you mean, okay let's leave it like this then. Just use `bridge.ns_exec` instead.